.changeset/nasty-jokes-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": minor
+---
+
+Add all legacy components
```

Comment:
Is this going into `main` or another branch? If to `main`, you'll want to update this :)

Reply:
It's going to `main`, but what does this need to be updated to?

Reply:
You're only adding 3 right? Or is this the branch that others will merge into?

Reply:
Ah yeah, this is the branch the others will merge into.